### ISSUE TITLE
Error creating account after invitation

### ISSUE THREAD

**Issue by evakq8r** [dani-garcia/bitwarden_rs#1071]
Issue: Unable to create a new Bitwarden account after invitation from the admin panel, due to error: **Error converting value "_" to type 'System.Nullable`1[System.Guid]'. Path 'organizationUserId', line 1, position 3242.**

### Environment
* Bitwarden_rs version: 1.16.0
* Install method: Unraid plugin
* Reverse proxy and version: Using the latest version of linuxserver/letsencrypt

### Steps to reproduce
* Changed default port number to 8085 (to avoid conflict with other docker containers)
* SIGNUPS_ALLOWED: true (was false, enabled as a test but doesn't change outcome)
* INVITATIONS_ALLOWED: true
* Added the LOG_FILE, LOG_LEVEL and EXTENDED_LOGGING variables from https://github.com/dani-garcia/bitwarden_rs/wiki/Logging, albeit the file/logs do not appear to be being generated.
* All other settings are default
* Logged into admin panel, setup SMTP settings, invited 2x users, received email with invite (points to an external domain address sitting behind my reverse proxy), attempt to create account after acknowledging invite.

### Expected behaviour
* Accounts created after invite received without error

### Actual behaviour
* Pressing Submit failed within a couple of seconds and received the error at the top of this issue.

### Relevant logs
* Unable to find any logs within the bitwarden installation folder (default for Unraid seems to be /mnt/user/appdata/bitwarden, no log files available).
* Screenshot below after filling info in (screenshot has info removed for privacy, but all fields bar the optional password hint field are filled in normally)

![image](https://user-images.githubusercontent.com/14250328/88455225-c3a39680-ceb2-11ea-8f07-8033f67e6773.png)

**Comment by dani-garcia** [comment]
That error looks from C#, are you sure you are using bitwarden_rs and not the official server?

**Comment by evakq8r** [comment]
Yep, definitely bitwarden_rs (URL in the screenshot redirects to https://hub.docker.com/r/bitwardenrs/server/):

![image](https://user-images.githubusercontent.com/14250328/88491862-90adef80-cfe5-11ea-9c9a-e9f83a07a713.png)

That's from the Apps store within Unraid.

**Comment by evakq8r** [comment]
Closing this issue as I've worked around it with the official Bitwarden install on Unraid. While not in the App store in Unraid, managed to change the docker-compose to sit on one network and have them on a reverse proxy, which thus far has worked well. Not sure it's worth while spending time @dani-garcia on looking into this, but if you and want me to re-install, happy to try and work out why the error comes up.
